### DISCUSSION TITLE
Responsiveness on different devices when displaying two months.

### DISCUSSION THREAD

**Discussion by bastekpew** [gpbl/react-day-picker#1707] [Support] [answered]
Hello,
I updated DayPicker from version 7 to 8 and set it to display 2 months. It looks fine on desktop but on phones and smaller screens, the months are displayed next to each other, not below each other like in version 7. Is there a way to change this? Thank you.
![0deb401b9ffe8e1df2f1cc5ba480b12 (1)](https://user-images.githubusercontent.com/36380670/222240586-d6245b26-3264-4dff-b1da-36b1cb6b9ff0.jpg)

**Comment by gpbl** [comment, marked as answer]
Yes, in v8 we removed this responsive behavior because it's better to be controlled by the developer. You can alter the behavior of the container via CSS, e.g.:

```css
.rdp-months {
  flex-wrap: wrap;
}
```

https://codesandbox.io/s/rdp-flex-wrap-hgkhqj?file=/src/styles.css:161-195

See [Styling DayPicker](https://react-day-picker.js.org/basics/styling) to read how to style the component.
